src/components/LoginView.vue

```
@@ -101,7 +101,7 @@ const testConnection = async () => {
     pingResult.value = 'Connection successful';
     pingSuccess.value = true;
   } catch (error) {
-    pingResult.value = 'Connection failed: ' + error.message;
+    pingResult.value = 'Connection failed: ' + (error as any).message;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)